Source/WebCore/style/StyleAdjuster.cpp

```diff
@@ -962,6 +962,12 @@ void Adjuster::adjustForSiteSpecificQuirks(RenderStyle& style) const
             && const_cast<Element*>(m_element.get())->classList().contains(class2))
             style.setMinHeight(WebCore::Length(0, LengthType::Fixed));
     }
+    if (m_document->quirks().needsPrimeVideoUserSelectNoneQuirk()) {
+        static MainThreadNeverDestroyed<const AtomString> className("webPlayerSDKUiContainer"_s);
+        if (m_element.get()->hasClassName(className))
```

Comment:
`RefPtr::operator->()` returns the `T*` it contains anyway
```suggestion
        if (m_element->hasClassName(className))
```